### ISSUE TITLE
AttributeError: module 'time' has no attribute 'clock'

### ISSUE THREAD

**Issue by Pixelao** [pyqtgraph/pyqtgraph#1255]
### Short description
The function time.clock() has been removed, after having been deprecated since Python 3.3.

AttributeError: module 'time' has no attribute 'clock' 

```
Traceback (most recent call last):
  File "c:/Users/Documents/Python_programs/Varios/basicgui.py", line 13, in <module>
    import pyqtgraph as pg
  File "C:\Users\AppData\Local\Programs\Python\Python38\lib\site-packages\pyqtgraph\__init__.py", line 204, in <module>
    from .graphicsItems.VTickGroup import *
  File "C:\Users\AppData\Local\Programs\Python\Python38\lib\site-packages\pyqtgraph\graphicsItems\VTickGroup.py", line 7, in <module>
    from .. import functions as fn
  File "C:\Users\AppData\Local\Programs\Python\Python38\lib\site-packages\pyqtgraph\functions.py", line 17, in <module>
    from . import debug
  File "C:\Users\AppData\Local\Programs\Python\Python38\lib\site-packages\pyqtgraph\debug.py", line 11, in <module>
    from . import ptime
  File "C:\Users\AppData\Local\Programs\Python\Python38\lib\site-packages\pyqtgraph\ptime.py", line 24, in <module>
    cstart = systime.clock()  ### Required to start the clock in windows
AttributeError: module 'time' has no attribute 'clock'
```

### Tested environment

 * PyQtGraph version: 0.10.0
 * Python version: **3.8.3 64-bit**
 * NumPy version: 1.18.4
 * Operating system: Windows 10 
 * Installation method: pip

**Comment by j9ac9k** [comment]
Pretty sure this was fixed in 0.11; which was released about a week ago. 

I'm going to close out the issue, but if it's still a problem, post back here and I'll reopen.

**Comment by Pixelao** [comment]
> Pretty sure this was fixed in 0.11; which was released about a week ago.
> 
> I'm going to close out the issue, but if it's still a problem, post back here and I'll reopen.

True, updated to 0.11 and working perfectly.

Thanks!
